### ISSUE TITLE
Consistent colour schemes

### ISSUE THREAD

**Issue by lukechurch** [larksystems/katikati-reporting#45]
![Screenshot 2020-08-10 at 09 52 55](https://user-images.githubusercontent.com/2978182/89765986-49e4ed00-daef-11ea-99ed-654cb9aa7aef.png)

In this chart, colours mean different things at different times - sometimes blue is incoming, sometimes its outgoing, sometimes its auto. Let's assign colours to one meaning throughout the charts.

For now this can be done manually in the config (I assume?)

**Comment by elayabharath** [comment]
Yep this can be added in the config - I have updated the projects. Could you take a look and let me know if you are okay with the color scheme? @lukechurch
